docs/articles_en/openvino_workflow/gen_ai.md

```diff
@@ -0,0 +1,165 @@
+
+.. code-block:: python
+
+    model.save_pretrained(model_dir)
+
+
+Alternatively, you can download and convert model using CLI interface: ``optimum-cli export openvino --model meta-llama/Llama-2-7b-chat-hf llama_openvino``
+In this case, you can load the converted model in OpenVINO representation directly from the disk:
+
+.. code-block:: python
+
+    model_id = "llama_openvino"
+    model = OVModelForCausalLM.from_pretrained(model_id)
+
+
+By default, inference will run on CPU. To select a different inference device, for example GPU, add `device="GPU"` to the `.from_pretrained()` call. To switch to a different device after the model has been loaded, use the `.to()` method. The device naming convention is the same as in OpenVINO native API:
+
+.. code-block:: python
+
+    model.to("GPU")
+
+
+Optimum-Intel API also provides out-of-the-box model optimization through weights compression using NNCF which substantially reduces the model footdpring and inference latency:
+
+.. code-block:: python
+
+    model = OVModelForCausalLM.from_pretrained(model_id, export=True, load_in_8bit=True)
+
+
+Wieght compression is applied by default to models larger than 1 billion parameters and also available for CLI interface as ``--int8`` option.
+
+Working with models tuned with LoRA
+++++++++++++++++++++++++++++++++++++
+
+Low-ranking Adaptation (LoRA) is a popular method to tune Generative AI models to a downstream task or custom data. But it requires some extra steps to be done for efficient deployment using the Hugging Face API. Namely, the trained adapters should be fused into the baseline model to avoid extra computation. This is how it can be done for LLMs:
+
+.. code-block:: python
+
+    model_id = "meta-llama/Llama-2-7b-chat-hf"
+    lora_adaptor = "./lora_adaptor"
+
+    model = AutoModelForCausalLM.from_pretrained(model_id, use_cache=True)
+    model = PeftModelForCausalLM.from_pretrained(model, lora_adaptor)
+    model.merge_and_unload()
+    model.get_base_model().save_pretrained("fused_lora_model")
+
+
+Now the model can be converted to OpenVINO using Optimum-Intel Python API or CLI interfaces mentioned above.
+
+Running Generative AI models using native OpenVINO APIs 
+########################################################
+
+To run Generative AI models using native OpenVINO APIs you need to follow regular **Сonvert -> Optimize -> Deploy** path with few simplifications. 
+
+To convert model from Hugging Face you can use Optimum-Intel export feature that allows to export model in OpenVINO format without invoking conversion API and tools directly, as it is shown above. In this case, the conversion process is a bit more simplified. You can still use regular conversion path if model comes from outside of Hugging Face ecosystem, i.e., in source framework format (PyTorch, etc.) 
+
+Model optimization could be performed within Hugging Face or directly using NNCF as described :doc:`here <weight_compression>`.
```

Comment:
```suggestion
Model optimization can be performed within Hugging Face or directly using NNCF as described in the :doc:`weight compression guide <weight_compression>`.
```

Reply:
Link to HF docs for model optimization: https://huggingface.co/docs/optimum/intel/optimization_ov (weights compression will be added soon)